Problems with Windows build following BULDING.md

**Describe the bug**

I carefully followed the Windows build instructions in [BUILDING.md](https://github.com/audacity/audacity/blob/master/BUILDING.md) but cannot get a successful build. I also spotted a couple of issues.

The *prequisites* section says "A working C++ 17 compiler" but that is platform specific. With `VS 2019` as specified the `cl.exe` version is 19

The instructions to install `conan` are `pip install conan` but they did not work as pip is not on the path. Rather, I used `py -m pip install conan` as specified somewhere else in the Audacity dev docs.

I installed the following tools:

- VS community 2019 (AKA 16 )
- cmake 3.2.2
- python 3.10.0

I chose the following options in `cmake` GUI:

- source `c:\projects\audacity\` - where I cloned `Audacity-3.1.2` too
- VS 16 2019
- x64 (default)
- native compilers

Get error " conan executable not found" even though I installed it as described.

```
Make Error at cmake-proxies/cmake-modules/conan.cmake:795 (message):
  Conan executable not found! Please install conan.
Call Stack (most recent call first):
  cmake-proxies/cmake-modules/conan.cmake:841 (conan_check)
  cmake-proxies/cmake-modules/AudacityDependencies.cmake:6 (conan_add_remote)
  CMakeLists.txt:175 (include)
```

![image](https://user-images.githubusercontent.com/618922/142723790-0c04f3d7-8729-49ac-bae7-6895f17665c4.png)

The `CMAKE_INSTALL_PREFIX` value seems odd for a dev build. I di not set it.

So I uncheck `audacity_conan_enabled` and try again

Get error ZLIB not found. The only value i can set for this option is `system` and Windows doesn't have a system version. Previous builds used the wx version of zlib and peeking at the cmake config files it looks like that should still be the case.

```
CMake Error at cmake-proxies/cmake-modules/AudacityDependencies.cmake:186 (message):
  Failed to find the system package ZLIB
Call Stack (most recent call first):
  cmake-proxies/CMakeLists.txt:1 (add_conan_lib)
```

![image](https://user-images.githubusercontent.com/618922/142724005-4a4f8605-a42e-495f-8eff-08f868d921ea.png)

I tried using the `cmake` command in CLI with same results (I'm working towards updating my [automated build script for asio](https://gist.github.com/SteveALee/da24c2be633340b8791066dd98eb5d0b) )

Any help much appreciated

thanks.

--- a/BUILDING.md
+++ b/BUILDING.md
@@ -7,6 +7,8 @@
 * **cmake** >= 3.16
 * A working C++ 17 compiler
 
+For Windows see below for important installer settings.
+
 ### Conan
 
 [The best way to install Conan is `pip`.](https://docs.conan.io/en/latest/installation.html)
@@ -27,7 +29,7 @@ Alternatively, on macOS, Conan is available from `brew`.
 
 ### CMake
 
-On Windows, please use the [prebuilt binaries](https://cmake.org/download/).
+On Windows, please use the [prebuilt binaries](https://cmake.org/download/). Ensure you select one of the options to add cmake to the system path.
 
 On macOS, the easiest way to install CMake is `brew install cmake`.
 
@@ -58,7 +60,9 @@ $ sudo apt-get install libgtk2.0-dev libasound2-dev libavformat-dev libjack-jack
 
 ## Building on Windows
 
-1. Clone Audacity from the Audacity GitHub project. 
+1. Ensure the Python installer option `Add python to environment variables` is checked. Go to Windows Settings "Add or Remove Programs" and modify Python settings if required.
+  
+2. Clone Audacity from the Audacity GitHub project. 
   
    For example, in the **git-bash** run:
 
@@ -85,6 +89,10 @@ $ sudo apt-get install libgtk2.0-dev libasound2-dev libavformat-dev libjack-jack
 Generally, steps 1-5 are only needed the first-time you configure. Then, after you've generated the solution, you can open it in Visual Studio next time. If the project configuration has changed, the IDE will invoke CMake internally. 
 
 > Conan Center provides prebuilt binaries only for **x64**. Configuring the project for Win32 will take much longer, as all the 3rd party libraries will be built during the configuration.
+
+### Building with ASIO support on Windows
+
+To enable ASIO support, please select `audacity_has_asio_support=On` in CMake after the intial configuration and then run select **Configure** again as described above. ASIO is only supported on Windows and only for 64-bit builds.
 
 ## macOS
 
@@ -155,11 +163,6 @@ cmake -GXcode -T buildsystem=1 -Daudacity_use_mad="off" -Daudacity_use_id3tag=of
 ### CMake options
 
 You can use `cmake -LH` to get a list of the options available (or use CMake GUI or `ccmake`). The list will include documentation about each option. For convenience, [here is a list](CMAKE_OPTIONS.md) of the most notable options.
-
-### Building with ASIO support on Windows
-
-To enable ASIO support on Windows, please pass `audacity_has_asio_support=On` to CMake during the configuration. 
-ASIO is only supported on Windows and only for 64-bit builds.
 
 ### Building using system libraries
 

